leakcanary-analyzer/src/main/java/com/squareup/leakcanary/ShortestPathFinder.java

```diff
@@ -176,6 +176,13 @@ private void visitRootObj(LeakNode node) {
     RootObj rootObj = (RootObj) node.instance;
     Instance child = rootObj.getReferredInstance();
 
+    //We should ignore leaks where an android.os.Binder is the root of the leak.
+    //When you bind and unbind from a Service, the OS will tend to keep a reference to the Binder
```

Comment:
Not precise enough. It's not that the process tends to keep references. It just does, as long as the client binder isn't garbage collected. So clients can create temporary leaks until they are GCed.